mpas_analysis/config.default

```diff
@@ -322,7 +322,7 @@ baseDirectory = /dir/to/seaice/reference
 
 # directory where sea ice observations are stored
 baseDirectory = /dir/to/seaice/observations
-concentrationSH = Altiberg/Altiberg_1991-2017.nc
+concentrationAltibergSH = Altiberg/Altiberg_1991-2017.nc
```

Comment:
I think the corresponding name change didn't happen in the climatology map task.